cmd/svcat/command/namespaced.go

```diff
@@ -17,22 +17,22 @@ limitations under the License.
 package command
 
 // Namespaced is the base command of all svcat commands that are namespace scoped.
```

Comment:
can you fix teh godoc here? `Namespaced` => `NamespacedCommand`

Reply:
Fixed